packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -363,15 +369,18 @@ const Services = WebexPlugin.extend({
    * @property {string} SendUserActivationPTO.email - The email of the user.
    * @property {string} SendUserActivationPTO.reqId - The activation requester.
    * @property {string} SendUserActivationPTO.token - The client auth token.
+   * @property {object} SendUserActivationPTO.activationOptions - Extra options to pass when sending the activation.
    */
 
   /**
    * Send a request to activate a user using a client token.
    *
    * @param {SendUserActivationPTO} - The Parameter transfer object.
-   * @returns {AtlasDTO} - The DTO returned from the **Atlas** service.
+   * @returns {LicenseDTO} - The DTO returned from the **License** service.
    */
-  sendUserActivation({email, reqId, token}) {
+  sendUserActivation({
+    email, reqId, token, activationOptions
```

Comment:
new line params for readability
```suggestion
    email,
    reqId,
    token,
    activationOptions
```